src/pages/messaging/agent-chat-api/rtm-pushes/index.mdx

```diff
@@ -719,6 +719,7 @@ Based on the received `reason` we suggest a different reaction.
 | Type                                | Notes                                                                                                                                  | Suggested behavior                              |
 | ----------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------- |
 | `agent_disconnected_by_server`      | Agent has been disconnected by the server.                                                                                             | Don't reconnect.                                |
+| `agent_logged_out_remotely`         | Agent has been logged out remotely.                                                                                                    | Don't reconnect.                                |
```

Comment:
Hello @marcindebski :) I would appreciate it if you could take a look if the changes discussed on [Slack](https://livechat.slack.com/archives/C2FAU97K2/p1656422573291569?thread_ts=1656418375.658909&cid=C2FAU97K2) are introduced appropriately. I will tag you in the rest of the changes as I have two separate issues included in this PR. Thanks in advance :)